### PR TITLE
spec: enable RWL/SSD caches

### PR DESCRIPTION
**Pull request by lixiaoy1**
Signed-off-by: Li, Xiaoyan <xiaoyan.li@intel.com>

## Checklist
- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

---

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`

</details>

### CODE REVIEW

**Comment by lixiaoy1** [comment]
@dillaman Please have a look.

File: ceph.spec.in [hunk 1]
```diff
@@ -62,6 +62,8 @@
 %endif
 %bcond_with seastar
 %bcond_with jaeger
+%bcond_with rbd_rwl
```
Comment (dillaman) [inline ceph.spec.in:65]: I know that (historically) the cmake option is just `RBD_RWL`, but might want to be more specific that it's a cache here for clarity.

```suggestion
%bcond_with rbd_rwl_cache
```

Comment (lixiaoy1) [reply ceph.spec.in]: Updated, and thank you.

**Review by dillaman — APPROVED** [review]
lgtm -- thx

**Pull request merged**